docs/modules/ROOT/pages/references/functional-requirements.adoc

```diff
@@ -0,0 +1,91 @@
+= Functional Requirements
+
+This page discusses functional requirements.
+It currently mainly targets the Control API and the {product} Portal.
+
+== Feature: Authentication
+
+> As a xref:references/glossary.adoc#_user[User] +
```

Comment:
I think we should start out with separate pages right from the beginning, there will be more to come quite soon I guess.
Rename this file to /references/functional-requirements/portal.adoc
```suggestion
= Functional Requirements Targeting the Portal

This page discusses functional requirements targeting the Control API and the {product} Portal.

== Feature: Authentication

> As a xref:references/glossary.adoc#_user[User] +
```